CHANGELOG.next.asciidoc

```diff
@@ -122,6 +117,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 - Fix race condition in http monitors using `mode:all` that can cause crashes. {pull}29697[pull]
 - Fix broken ICMP availability check that prevented heartbeat from starting in rare cases. {pull}29413[pull]
+- Fix broken macOS ICMP python e2e test. {pull}29900[29900]
+- Only add monitor.status to browser events when summary. {pull}29460[29460]
+- Also add summary to journeys for which the synthetics runner crashes. {pull}29606[29606]
+- Update size of ICMP packets to adhere to standard min size. {pull}29948[29948]
+- Add fonts to support more different types of characters for multiple languages. {pull}29606[29861]
```

Comment:
```suggestion
- Add fonts to support more different types of characters for multiple languages. {pull}29861[29861]
```

Seems like this was the right link but wrong named reference:

See: https://github.com/elastic/beats/pull/29861